symengine/tests/basic/test_arit.cpp

```diff
@@ -559,6 +559,15 @@ TEST_CASE("Pow: arit", "[arit]")
     r2 = zero;
     REQUIRE(eq(*r1, *r2));
 
+    r1 = pow(zero, zero);
+    REQUIRE(eq(*r1, *one));
+
+    r1 = pow(zero, i2);
+    REQUIRE(eq(*r1, *zero));
+
+    r1 = pow(zero, im1);
+    REQUIRE(r1->__str__() == "zoo");
```

Comment:
Can you change this to `eq(*r1, *ComplexInf)`? (You might need to add `using SymEngine::ComplexInf` to the top